app/assets/stylesheets/all/components/_form_items.scss

```diff
@@ -99,15 +100,9 @@
     }
 
     .actions {
-      text-align: right;
+      text-align: end;
```

Comment:
This is cool but not sure how well supported:

https://caniuse.com/#feat=css-logical-props

You can use `$right` though. That will be `right` for LTR and `left` for RTL. Try it?